cernopendata/modules/fixtures/data/records/atlas-ATL-PHYS-PUB-2022-039.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "abstract": {
+      "description": "<p>FIXME This dataset serves for performance studies of constituent-based jet taggers on large radius boosted top quark jets reconstructed from optimized jet input objects in simulated collisions at √ 𝑠 = 13 TeV. Several taggers which consider all the information contained in the kinematic information of the jet constituents are tested, and compared to a tagger which relies on high-level summary quantities similar to the taggers used by ATLAS in Runs 1 and 2. Several constituent based taggers are found to out-perform the high level quantity based tagger, with the best achieving a factor of two increase in background rejection across the kinematic range.</p> <p>FIXME This dataset consists of two files, the training data and the test data, both available in H5 format. The dataset semantics and code usage examples are linked below.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "ATLAS collaboration"
+    },
+    "collections": [
+      "ATLAS-Derived-Datasets"
+    ],
+    "dataset_semantics": [
+      {
+        "description": "FIXME Transverse momentum of the jet.",
+        "type": "Float_t",
+        "variable": "FIXME jetPt"
+      },
+      {
+        "description": "FIXME Transverse momentum of the jet.",
+        "type": "Float_t",
+        "variable": "FIXME jetPt"
+      }
+    ],
+    "date_published": "2022",
+    "distribution": {
+      "formats": [
+        "h5"
+      ],
+      "number_events": 0,
+      "number_files": 2,
+      "size": 138693229336
+    },
+    "doi": "10.7483/OPENDATA.ATLAS.FG5F.96GA",
+    "experiment": "ATLAS",
+    "files": [
+      {
+        "checksum": "adler32:01126419",
+        "size": 8157858156,
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/datascience/ATL-PHYS-PUB-2022-039/test.h5"
+      },
+      {
+        "checksum": "adler32:07ff5788",
+        "size": 138693229336,
+        "uri": "root://eospublic.cern.ch//eos/opendata/atlas/datascience/ATL-PHYS-PUB-2022-039/train.h5"
+      }
+    ],
+    "keywords": [
+      "datascience"
+    ],
+    "links": [
+      {
+        "description": "ATLAS publication note ATL-PHYS-PUB-2022-039",
+        "url": "http://cds.cern.ch/record/2825328/files/ATL-PHYS-PUB-2022-039.pdf"
+      }
+    ],
+    "publisher": "CERN Open Data Portal",
+    "recid": "15013",
+    "title": "A dataset for constituent-based top-quark tagging studies in the ATLAS detector",
```

Comment:
```suggestion
    "title": "LHC Machine Learning Challenge: The Hunt for Boosted Top Quarks",
```

Reply:
Maybe we can use a more interesting title?

Reply:
Such a title may be too general in my eyes, e.g. ATLAS is not even mentioned there.

If I compare to the past ATLAS Kaggle challenge, which was also released on the CERN Open Data portal ([see here](https://opendata.cern.ch/search?page=1&size=20&q=kaggle)), we had a dataset record, a software record, and a documentation record for the challenge. While here we only have the dataset record. 

Looking in your updated abstract, you used the expression "ATLAS Top Tagging Open Data Set", so I reused it also for the title.  It may be better perhaps? Although it may still be perhaps a bit too general, in case there are more top tagging ML studies coming from ATLAS in the next few years.

Anyway please make a choice within ATLAS and please let me know which title would you prefer. It may be then good to keep it also in GitLab repository etc the same, for consistency.

Reply:
I think what you have written now is likely the best compromise. It is the title of the gitlab repository, which has undergone a small collaboration review. I think it makes sense for the two to match.